tests/default_test.ts

```diff
@@ -151,3 +151,12 @@ test(
     await assertExpectedFiles();
   },
 );
+
+test(
+  { fixture: "terraform/terraform-project", args: ["--no-strict"] },
```

Comment:
The proposed example has a detectable Terraform version, `--no-strict` flag isn't required:
```suggestion
  { fixture: "terraform/terraform-project", args: [] },
```